src/layout/Grid.js

```diff
@@ -65,23 +69,24 @@
         rowSpan = rowSpan || 1;
         colSpan = colSpan || 1;
         title = title || "";
+        var mult = this.cellDensity() ? this.cellDensity() : 1;
```

Comment:
The condition on that line is probably not needed. It's just intended to allow cellDensity = null or cellDensity = 0